### DISCUSSION TITLE
"prefer-effect-callback-in-block-statement" vs. "arrow-body-style"

### DISCUSSION THREAD

**Discussion by timplateus** [timdeschryver/eslint-plugin-ngrx#266] [General]
How would you solve the conflict between the ngrx/prefer-effect-callback-in-block-statement and [eslint/arrow-body-style](https://eslint.org/docs/rules/arrow-body-style)?

**Comment by timdeschryver** [comment]
AFAIK there's no way to enable both of them.
You can disable the ngrx rule.
Or create a specific entry in the eslint config to look for `*.effects.ts` files instead of `*.ts` where you disable the eslint arrow body rule.

**Reply by timplateus** [reply]
Disabling the eslint rule on *.effects.ts looks like the most efficient solution indeed. Thanks!
